cores/psoc/main.cpp

```diff
@@ -21,33 +21,51 @@
 #include "cyhal.h"
 #include "cybsp.h"
 #include "cy_retarget_io.h"
+
+#include <FreeRTOS.h>
+#include <task.h>
+
 #include "time.h"
 
+#define ARDUINO_MAIN_TASK_STACK_SIZE    (4096u)
+#define ARDUINO_MAIN_TASK_PRIORITY      (2u)
```

Comment:
How is the priority assigned?

Reply:
A bit arbitrary. In principle this is the main thread, so this should have a high prio. But not necessarily the highest, as the threads from lwip need to take over (some have 0 prio), and they are not necessarily permanent.
But the thread config is not yet very thoughtfully dimensioned 😅